packages/lit-dev-tools-esm/src/dev-server.ts

```diff
@@ -16,7 +16,8 @@ import {createGitHubTokenExchangeMiddleware} from 'lit-dev-server/lib/middleware
 import {dev as ENV} from 'lit-dev-tools-cjs/lib/lit-dev-environments.js';
 
 const THIS_DIR = pathlib.dirname(fileURLToPath(import.meta.url));
-const CONTENT_PKG = pathlib.resolve(THIS_DIR, '..', '..', 'lit-dev-content');
+const REPO_ROOT = pathlib.resolve(THIS_DIR, '..', '..', '..');
+const CONTENT_PKG = pathlib.resolve(REPO_ROOT, 'packages', 'lit-dev-content');
```

Comment:
node resolve would be good here too

Reply:
Done in https://github.com/lit/lit.dev/pull/863